DQMOffline/Trigger/python/TopMonitoring_cff.py

```diff
@@ -608,19 +850,27 @@
     + fullyhadronic_TripleBTag_all
     + fullyhadronic_TripleBTag_jet
     + fullyhadronic_TripleBTag_bjet
-    + topDiElectronHLTMonitor_Dz
-    + topElecMuonHLTMonitor_Dz_Mu12Ele23
-    + topElecMuonHLTMonitor_Dz_Mu8Ele23
-    + topElecMuonHLTMonitor_Dz_Mu23Ele12
-    + topElecMuonHLTMonitor_Mu12Ele23
-    + topElecMuonHLTMonitor_Mu8Ele23
-    + topElecMuonHLTMonitor_Mu23Ele12
-    + topElecMuonHLTMonitor_Mu12Ele23_ref
-    + topElecMuonHLTMonitor_Mu8Ele23_ref
-    + topElecMuonHLTMonitor_Mu23Ele12_ref
-    + topDiMuonHLTMonitor_Dz_Mu17_Mu8,
-     cms.Task(egmGsfElectronIDsForDQM) # Use of electron VID requires this module being executed first
+
+    , cms.Task(egmGsfElectronIDsForDQM) # Use of electron VID requires this module being executed first
 )
 
 topHLTDQMSourceExtra = cms.Sequence(
 )
+
+topMonitorHLT_2016 = topMonitorHLT.copy()
+topMonitorHLT_2016.remove( topEleJet_jet )
+topMonitorHLT_2016.remove( topEleJet_ele )
+topMonitorHLT_2016.remove( topEleJet_all )
+topMonitorHLT_2016.remove( topEleHT_ht )
+topMonitorHLT_2016.remove( topEleHT_ele )
+topMonitorHLT_2016.remove( topEleHT_all )
+topMonitorHLT_2016.remove( topDiMuonHLTMonitor_Mass3p8 )
+topMonitorHLT_2016.remove( topDiMuonHLTMonitor_Mass8Mon )
+topMonitorHLT_2016.remove( topDiMuonHLTMonitor_Mass3p8Mon )
+topMonitorHLT_2016.remove( fullyhadronic_ref370 )
+topMonitorHLT_2016.remove( fullyhadronic_ref430 )
+topMonitorHLT_2016.remove( fullyhadronic_TripleBTag_all )
+topMonitorHLT_2016.remove( fullyhadronic_TripleBTag_jet )
+topMonitorHLT_2016.remove( fullyhadronic_TripleBTag_bjet )
+
+#run2_HLTconditions_2016.toReplaceWith(topMonitorHLT, topMonitorHLT_2016)
```

Comment:
I guess this last line was commented because it would be picked up when running `cmsDriver.py` with `--era Run2_2017` and `--era Run2_2018` (which is not what's intended), since the latter Eras seem to include the `run2_HLTconditions_2016` modifier.

One solution would be to remove the `run2_HLTconditions_2016` modifier from `Run2_2017` (and the `run2_HLTconditions_2017` modifier from `Run2_2018`), similarly to what was done in #26489 for the `run2_muon_201*` modifiers; then, this last line could be uncommented to have the right sequence for 2016.

If the reviewers agree with this change to `Era_Run2_2017_cff` and `Era_Run2_2018_cff`, you could include it directly in this PR @skonstantinou